otherlibs/unix/unix.mli

```diff
@@ -601,11 +621,11 @@ val fchmod : file_descr -> file_perm -> unit
 
 val chown : string -> int -> int -> unit
 (** Change the owner uid and owner gid of the named file.
-    On Windows: not implemented (make no sense on a DOS file system). *)
+    On Windows: not implemented. *)
```

Comment:
This is very much up to you, and only affects unixLabels.mli. The formatting of `On Windows` comments is inconsistent. I think that the nice version is: `On Windows: whatever.` (as here) rather than `On Windows, ` (in some other places) but also with a blank line before it so it becomes `<p>On Windows: whatever.</p>` in the final output (that's not the case here, but it is in other places).

Reply:
All fixed.